code/modules/power/cable.dm

```diff
@@ -166,14 +172,19 @@ GLOBAL_LIST_INIT(wire_node_generating_types, typecacheof(list(/obj/structure/gri
 		return
 
 	else if(W.tool_behaviour == TOOL_MULTITOOL)
-		if(powernet && (powernet.avail > 0)) // is it powered?
-			to_chat(user, "<span class='danger'>Total power: [DisplayPower(powernet.avail)]\nLoad: [DisplayPower(powernet.load)]\nExcess power: [DisplayPower(surplus())]</span>")
-		else
-			to_chat(user, "<span class='danger'>The cable is not powered.</span>")
+		to_chat(user, get_power_info())
 		shock(user, 5, 0.2)
 
 	add_fingerprint(user)
 
+
+/obj/structure/cable/proc/get_power_info()
+	if(powernet && (powernet.avail > 0)) // is it powered?
```

Comment:
```suggestion
	if(powernet?.avail > 0)
```

Comment is not necessary--comments are for why, not what.

Reply:
I know it's not your code, but it's good to get it out of the way now.

Reply:
wouldn't it be nice to keep the comment?